### DISCUSSION TITLE
关于上传图片的小tip...

### DISCUSSION THREAD

**Discussion by warning-Kirov-reporting** [share121/inter-knot#930] [其他]
上传图片后别太快发送，不然就会像这位不守规矩的治安官一样上传失败。
没错，图也是偷的！
![屏幕截图 2024-08-10 191206](https://github.com/user-attachments/assets/3c4eb41c-7501-4f12-9c4b-fc1fa5a3a7c1)
![image](https://github.com/user-attachments/assets/85afac17-5cbd-43d3-a5e4-940b374fc34e)

**Comment by capacitor1** [comment]
支持

**Comment by SixRadium** [comment]
嘿嘿，我就是那一个帖子的楼主

**Comment by ZhuYuan-148** [comment]
谢谢市民提醒…
